api/src/main/java/com/datastrato/gravitino/rel/TableChange.java

```diff
@@ -32,6 +33,8 @@
  */
 @Evolving
 public interface TableChange {
+  /** A default value that indicates the default value is not set. */
+  Expression DEFAULT_VALUE_NOT_SET = Column.DEFAULT_VALUE_NOT_SET;
```

Comment:
It's unnecessary to redefine `DEFAULT_VALUE_NOT_SET`, `Column.DEFAULT_VALUE_NOT_SET` can be reused directly